pkg/storegateway/bucket.go

```diff
@@ -1193,6 +1194,7 @@ func (s *BucketStore) recordSeriesCallResult(safeStats *safeQueryStats) {
 	s.metrics.cachedPostingsCompressedSizeBytes.Add(float64(stats.cachedPostingsCompressedSizeSum))
 	s.metrics.seriesHashCacheRequests.Add(float64(stats.seriesHashCacheRequests))
 	s.metrics.seriesHashCacheHits.Add(float64(stats.seriesHashCacheHits))
+	s.metrics.expandPostingsDuration.Observe(float64(stats.expandedPostingsDuration))
```

Comment:
Can we ensure we have at least 1 test asserting on these metrics? We can't assert on actual duration (it varies on each request) but we could at least ensure it gets tracked (value is `> 0`).

Reply:
I pushed https://github.com/grafana/mimir/pull/3661/commits/31c138181b57bfaa15ef351854774dc05a9d7427, which should address this